packages/bee-q/src/components/dropdown/_storybook/bq-dropdown.stories.tsx

```diff
@@ -0,0 +1,169 @@
+import type { Args, Meta, StoryObj } from '@storybook/web-components';
+
+import { html } from 'lit-html';
+
+import mdx from './bq-dropdown.mdx';
+
+import { PANEL_PLACEMENT } from '../../panel/bq-panel.type';
+
+const meta: Meta = {
+  title: 'Components/Dropdown',
+  component: 'bq-dropdown',
+  parameters: {
+    docs: {
+      page: mdx,
+    },
+    layout: 'centered',
+  },
+  argTypes: {
+    'panel-distance': { control: 'number' },
+    'panel-placement': { control: 'select', options: PANEL_PLACEMENT },
+    'panel-open': { control: 'boolean' },
+    'panel-scrollbar': { control: 'boolean' },
+    // Event handlers
+    bqPanelChange: { action: 'bqPanelChange' },
+    // Event handler (bq-option-list)
+    bqSelect: { action: 'bqSelect', table: { disable: true } },
+    // Not part of the public API, so we don't want to expose it in the
+    children: { control: 'text', table: { disable: true } },
+  },
+  args: {
+    'panel-distance': 0,
+    'panel-placement': 'bottom',
+    'panel-open': false,
+    'panel-scrollbar': false,
+    children: { control: 'text', table: { disable: true } },
+  },
+};
+export default meta;
+
+type Story = StoryObj;
+
+const Template = (args: Args) => {
+  return html`
+    <bq-dropdown
+      panel-distance=${args['panel-distance']}
+      panel-placement=${args['panel-placement']}
```

Comment:
You can use a direct return here:

```suggestion
const Template = (args: Args) => html`
    <bq-dropdown
      panel-distance=${args['panel-distance']}
      panel-placement=${args['panel-placement']}
```